internal/services/cognitive/cognitive_deployment_resource.go

```diff
@@ -314,6 +323,10 @@ func (r CognitiveDeploymentResource) Update() sdk.ResourceFunc {
 				properties.Properties.Model.Version = pointer.To(model.Model[0].Version)
 			}
 
+			// if metadata.ResourceData.HasChange("dynamic_throttling_enabled") {
+			// }
+			properties.Properties.DynamicThrottlingEnabled = pointer.FromBool(model.DynamicThrottlingEnabled)
```

Comment:
Is there a reason this is commented out?

Reply:
no :sweat_smile: